lib/bindings/MQTTBinding.js

```diff
@@ -203,8 +248,9 @@ function sendConfigurationToDevice(apiKey, deviceId, results, callback) {
             apiKey,
             payload
         );
+    var leadingSlash = config.getConfig().mqtt.avoidLeadingSlash ? '' : '/';
```

Comment:
What about if config.getConfig().mqtt.avoidLeadingSlash is not defined? It will be the case of existing IOTA deployments in which case leading slashes should be included, due to backward compatibility.

Reply:
In that case `leadingSlash` would be `/`
( `undefined ? '' : '/' ` evaluates to `/`)

Reply:
Nice! NTC